src/emitter.rs

```diff
@@ -334,3 +385,29 @@ fn need_quotes(string: &str) -> bool {
         || string.parse::<i64>().is_ok()
         || string.parse::<f64>().is_ok()
 }
+
+/// Check if the string can be expressed a valid literal block scalar.
+/// The YAML spec supports all of the following in block literals except #xFEFF:
+///      #x9 | #xA | [#x20-#x7E]          /* 8 bit */
+///   | #x85 | [#xA0-#xD7FF] | [#xE000-#xFFFD] /* 16 bit */
+///   | [#x10000-#x10FFFF]                     /* 32 bit */
+fn is_valid_literal_block_scalar(string: &str) -> bool {
```

Comment:
```suggestion
/// The YAML spec supports all of the following in block literals except `#xFEFF`:
/// ```ignore
///     #x9 | #xA | [#x20-#x7E]                /* 8 bit */
///   | #x85 | [#xA0-#xD7FF] | [#xE000-#xFFFD] /* 16 bit */
///   | [#x10000-#x10FFFF]                     /* 32 bit */
/// ```
fn is_valid_literal_block_scalar(string: &str) -> bool {
```